### ISSUE TITLE
Needs an update

### ISSUE THREAD

**Issue by garkpit** [swlaschin/Railway-Oriented-Programming-Example#8]
Please could you update this to work with the latest versions of everything. I've spent ages (minutes!) trying to get it working in Visual Studio 2019, but as I don't really know what I am doing (first time ever with F# - it looks sweeeet) I've not had any luck. Many thanks.

**Comment by MatrixMike** [comment]
Yes I tried too ages ago. I think repos that are followed should be regularly flagged to be checked. At least we could more specifically detail the requirements of the repo. This is a common problem for me so I lend my support to any system that promotes this.

**Comment by swlaschin** [comment]
Sorry everyone! This repo is 6 years old now and a lot of of things have changed!.  Thanks for the reminder, I'll update it ASAP.
